public-ui/src/app/data-browser/views/genomic-view/genomic-view.component.tsx

```diff
@@ -113,6 +113,10 @@ export const GenomicViewComponent = withRouteData(class extends React.Component<
         this.resetFilters();
     }
 
+    componentWillUnmount() {
+        localStorage.setItem('genomicSearchText', '');
```

Comment:
Removing the search term here to make sure user does not see same results once they come back again.